client/src/components/Loading.tsx

```diff
@@ -0,0 +1,25 @@
+import { ScanSpinner } from 'react-craftify-spinners';
+import { IoMdArrowBack as Arrow } from 'react-icons/io';
+
+import Button from './Button';
+
+const Loading = ({ prev }: { prev: () => void }) => {
+  return (
+    <div className='max-w-48 m-auto'>
+      <Button
+        icon={Arrow}
+        variant='tertiary'
+        className='absolute left-16'
+        onClick={() => prev()}
+      >
+        Back
+      </Button>
+      <div className='flex flex-col items-center gap-3 text-center py-20'>
+        <ScanSpinner />
```

Comment:
now that i look at it, the ScanSpinner makes it looks like we're verifying someone/something instead of generating notes lol, is it misleading? but i think it's fine since we have the text saying that it's generating notes

Reply:
That's true lol I don't know if I like it so I might change it to be a generic spinner or just show a loading icon on the previous page instead of creating a whole new screen for that

Reply:
Actually I think it would be good to keep it as a separate loading screen. I saw some UI that had either the button have a loading state or a loading icon next to it, but that would cause other complications if the user interacts with the toggles or slider while it's generating